main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for nickname, player_data in players.items():
+        # Get or create the race
+        race, created = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
```

Comment:
I do not see that you using `created`  viriable somewhere, for unsued viriables you can use this `_`